src/main/scala/viper/silver/reporter/Reporter.scala

```diff
@@ -55,7 +55,7 @@ case class CSVReporter(name: String = "csv_reporter", path: String = "report.csv
         })
       case CopyrightReport(text) =>
       case MissingDependencyReport(text) =>
-
+      case BackendSubProcessReport(_, _, _, _) =>
```

Comment:
I suggest to combine the "noop" cases and add a comment explaining why there's nothing to do here. E.g. `case _: CopyrightReporter | _: MissingDependencyReport | _: BackendSubProcessReport => // Irrelevant for reporting`.